### PR TITLE
Fix broken cracklib macro in autotools and meson

### PR DESCRIPTION
**Pull request by dgsga**
WIP

### CODE REVIEW

File: meson.build [hunk 1]
```diff
@@ -1724,20 +1724,37 @@ sed_command = [
...
+    cracklib_dict += '/usr/share/cracklib/pw_dict'
+elif fs.is_dir('/var/cache/cracklib')
+    cracklib_dict += '/var/cache/cracklib/cracklib_dict'
+elif fs.is_dir('/usr/local/libdata/cracklib')
+    cracklib_dict += '/usr/local/libdata/cracklib/cracklib-words'
+elif fs.is_dir('/usr/pkg/share/cracklib')
+    cracklib_dict += '/usr/pkg/share/cracklib/pw_dict'
+elif fs.is_dir('/usr/local/share/cracklib')
+    cracklib_dict += '/usr/local/share/cracklib/cracklib-small'
+endif
```
Comment (rdmark) [inline meson.build:1739]: Holy moly, this is quite the collection. Did you dig through all the actively supported distros to collect these paths?

**Review by rdmark — APPROVED** [review]

File: meson.build (hunk 1 above)
Comment (dgsga) [reply meson.build:1739]: Of course... :)

**Comment by rdmark** [comment]
@dgsga Will you port this to 2.x or should I?

**Comment by dgsga** [comment]
All done in PR #990

**Pull request merged**